client/blindbox/requests/secure_session.py

```diff
@@ -11,12 +11,14 @@ class SecureSession(Session):
     def __init__(
         self,
         addr: str,
+        cce_file: str,
```

Comment:
There is no simulation mode, is that in purpose?

Reply:
Simulation is now the same as debug mode